staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1798,9 +1798,6 @@ const (
 	DeprecatedSlirpInterface NetworkInterfaceType = "slirp"
 	// Virtual machine instance masquerade interface
 	MasqueradeInterface NetworkInterfaceType = "masquerade"
-	// Virtual machine instance passt interface is deprecated
```

Comment:
@orelmisan I see that SLIRP and PASST have both the same comment:
```
// Deprecated: Removed in v1.3.
```
I don't understand why SLIRP comparing to PASST wasn't removed.

Reply:
Thank you for reviewing the PR @enp0s3.
SLIRP was discontinued after it was GA.
Passt was discontinued when it still had a feature gate protecting it.
Please see https://github.com/kubevirt/kubevirt/pull/13153#discussion_r1897142806.